src/documentation/transxchange_to_ntfs_specs.md

```diff
@@ -175,24 +176,42 @@ NTFS field | TransXChange element | Mapping rule/Comment
-drop_off_type | *JourneyPatternTimingLink/To/Activity* | `1` when the input value is `pickUp`, `0` otherwise.
+departure_time | *JourneyPatternTimingLink/RunTime*, *JourneyPatternTimingLink/From/WaitTime* | See computing rule above.
+stop_id | *JourneyPatternTimingLink/From/StopPointRef* | This field is prefixed. Link to the file [stops.txt](#stopstxt).
+stop_sequence | *JourneyPatternTimingLink/From{SequenceNumber}* | The value should be `1` for the first stoppoint of the trip.
+pickup_type | *JourneyPatternTimingLink/From/Activity* | `1` when the input value is `setDown`, `0` otherwise.
+drop_off_type | *JourneyPatternTimingLink/From/Activity* | `1` when the input value is `pickUp`, `0` otherwise.
+
+## Possible evolutions
+The following points might be possibly included in a later version of the connector:
 
-### comments.txt
-Only comments on trips are handled in the present version.
+### Provide comments for trips
+Two additional files for comments are to be created:
 
+**comments.txt**
 NTFS field | TransXChange element | Mapping rule/Comment
 --- | --- | ---
 comment_id | *VehicleJourney/VehicleJourneyCode*, *VehicleJourney/Note/NoteCode* | This field is prefixed and formed by the concatenation of the two fields separated by a `:`.
 comment_name | *VehicleJourney/Note/NoteText* |
 
-### comment_links.txt
-
+**comment_links.txt**
 NTFS field | TransXChange element | Mapping rule/Comment
 --- | --- | ---
 object_id | *VehicleJourney/ServiceRef*, *VehicleJourney/LineRef*, *VehicleJourney/VehicleJourneyCode* | This field is prefixed. Link to the file [trips.txt](#tripstxt).
 object_type |  | Fixed value `trip`.
 comment_id | *VehicleJourney/VehicleJourneyCode*, *VehicleJourney/Note/NoteCode* | This field is prefixed. Link to the file [comments.txt](#commentstxt).
+
+### Provide accessibility info for trips
+The wheelchair_accessible property is created in the trip_properties.txt file after the element *VehicleJourney/Operational/VehicleType/WheelchairAccessible*. The value of wheelchair_accessible is `1` when the trip is accesible, `2` when the trip is not accessible and `0` when the field is not specified.
+
+### Take into account exceptional service dates
+The current version handles only regular days and bank holidays that are explicitily specified as normal elements of an *OperatingProfile*. Special or periodic service dates (e.g. *SpecialDaysOfOperation*, *PeriodicDayType*, *ServicedOrganisationDayType*) are not yet handled, they might be added in a later version.
+
+### Handle shared attributes of *JourneyPattern* and *VehicleJourney*
+The current version does not take into account the common properties of a *VehicleJourney* shared with the associated *JourneyPattern*. In particular, waiting times or the activity at a timing link that are explicitly specified in the scope of a *VehicleJourneyTimingLink* are not yet handled and might be added in a later version.
```

Comment:
It's done and ok for VehicleJourneyTimingLink/Activity pickUp and setDown.
It just misses the "pass" Activity